models/user.js

```diff
@@ -1,11 +1,21 @@
 const database = require('../database/dbConfig');
+const UserRole = require("./user_role");
 
-const add = user => {
```

Comment:
I see on line 4 it shows (user, role_id=3) I am curious why we need that instead of just role_id.  i am assuming you did that for testing

Reply:
Oh right, that sets a default value for role in case one isn't given. This works assuming that the role_id for alumni will be 3, so I may actually want to change that to look for it by role name.